.circleci/config.yml

```diff
@@ -22,3 +31,12 @@ jobs:
       - run:
           name: Parse SassDoc comments
           command: npm run sassdoc
+workflows:
+  build_and_test:
+    jobs:
+      - test:
+          matrix:
+            parameters:
+              # https://github.com/CircleCI-Public/cimg-ruby
+              # only supports the last three ruby versions
+              ruby-version: ["2.5", "2.6", "2.7", "3.0", "3.1", "3.2"]
```

Comment:
Ruby 2.5 and 2.6 are End of Life I don't think we need to build ci against them https://endoflife.date/ruby but this looks great! Thanks for the PR

Reply:
👍 I'll pull them out.